### PR TITLE
Add support for Scala 2.13.9

### PR DESCRIPTION
**Pull request by henricook**
Fixes: https://github.com/scapegoat-scala/scapegoat/issues/679

This is my first contribution to this project so please let me know if anything is missing

Requires a release after merge as it updates the README

### CODE REVIEW

File: .github/workflows/pr-checks.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ jobs:
           - 2.13.6
```
Comment (saeltz) [inline .github/workflows/pr-checks.yml:16]: Could you remove this line? We’re only camping for the last three versions of 2.13.

File: build.sbt [hunk 2]
```diff
@@ -22,8 +22,8 @@ developers := List(
   )
 )
 
-scalaVersion := "2.13.8"
-crossScalaVersions := Seq("2.11.12", "2.12.15", "2.12.16", "2.13.7", "2.13.8")
+scalaVersion := "2.13.9"
+crossScalaVersions := Seq("2.11.12", "2.12.15", "2.12.16", "2.13.7", "2.13.8", "2.13.9")
```
Comment (saeltz) [inline build.sbt:26]: Also here please remove 2.13.6.

**Review by saeltz — COMMENTED** [review]
Thanks for opening the pull request!
I left two suggestions and then it should be good.

File: build.sbt (hunk 2 above)
Comment (henricook) [reply build.sbt:26]: Can't see the 2.13.6 on that one 👀

File: .github/workflows/pr-checks.yml (hunk 1 above)
Comment (henricook) [reply .github/workflows/pr-checks.yml]: Removed, thanks for the feedback

File: build.sbt (hunk 2 above)
Comment (saeltz) [reply build.sbt:26]: My bad.

**Review by saeltz — APPROVED** [review]

File: build.sbt (hunk 2 above)
Comment (mccartney) [inline build.sbt:26]: I think we have the tradition of keeping two minor versions per major version. Thus I suggest to remove `2.13.7`.

**Review by mccartney — APPROVED** [review]
LGTM, but please remove the `2.13.7`.

File: build.sbt (hunk 2 above)
Comment (mccartney) [reply build.sbt:26]: Ah. Now I can see @saeltz said we are hoarding 3 versions. That's fine. I must've remembered wrong.

Comment (saeltz) [reply build.sbt:26]: I wasn’t sure either. In the CI we had three, in SBT two. We can still clean it up afterwards.

**Pull request merged**